demo/cypress/integration/disclosure.spec.js

```diff
@@ -0,0 +1,27 @@
+describe('Disclosure', () => {
+  it('should allow toggling disclosure', () => {
+    cy.visitComponentUrl('disclosure/example');
+
+    assertClosed();
+
+    cy.findByRole('button', { name: /View/i })
+      .click()
+      .should('have.attr', 'aria-expanded', 'true');
```

Comment:
This uses https://testing-library.com/docs/cypress-testing-library/intro/ to do things like find elements by role.